test/test_wiki_trac_rst_convert.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+from wiki_trac_rst_convert import convert_content
+
+
+class TracRstToVanillaRst(unittest.TestCase):
+    """
+    Test conversion of content from Trac-flavored reStructuredText to
+    vanilla reStructuredText, that is supported by GitHub
+    """
+
+    def test_empty_adds_newline(self):
+        """Adds a newline at the end of an empty file"""
+        self.assertEqual(convert_content(''), '\n')
+
+    def test_removes_RST_wrapping(self):
+        """Removes the Trac RST markers"""
+        self.assertEquals(
+            convert_content('{{{#!rst}}}'),
+            '\n'
+        )
+        self.assertEquals(
+            convert_content('{{{\n#!rst\n}}}'),
+            '\n'
+        )
+
+    def test_does_not_strip_content(self):
+        """When using str.strip, it deletes too much. Prevent this."""
```

Comment:
why should a test care about low level str.srip usage?

The test description should be done at a much higher level.

For example:

Leading and trailing newlines that are wrapped inside the Trac RST markers are preserved.

------------

Also, I see a lot of code duplication for the pattern `self.assertEquals(convert_content(SOMETHING), expected)`

This is a great candidate for creating a helper `self.assertConvertedContent(expected, SOMETHING)`